components/match2/wikis/smite/get_match_group_copy_paste_wiki.lua

```diff
@@ -0,0 +1,100 @@
+---
```

Comment:
please follow the cleaner code style we go for with the copy paste module
see #3952 and #3957

Reply:
I have tried setup /dev one based on the one for Wild Rift in #3957  but encounter this error

`Lua error in Module:GetMatchGroupCopyPaste/wiki at line 34: attempt to concatenate upvalue 'INDENT' (a nil value).` 

i'm not sure if its because its requiring other PR or not, but in case you want to check (this is WR with only KDA removed, while OOS you can actually removed KDA parts from Wildrift too its never been used there) <https://liquipedia.net/smite/Module:GetMatchGroupCopyPaste/wiki/dev>

Reply:
you need to test it with dev enabled
you can use https://liquipedia.net/smite/Special:RunQuery/BracketCopyPaste/dev for testing